person.rb

```diff
@@ -17,7 +20,10 @@ def of_age?
   def can_use_services?
     of_age? || @parent_permission
   end
-  public :can_use_services?
+
+  def correct_name
+    @name
+  end
 end
 
 # person = Person.new(57, 'Anja')
```

Comment:
- [ ] [OPTIONAL] Kindly remove all the commented code in this project in order to have a clean and more readable code-base.

Reply:
Okay I will Thank you @Whoistolu for the code review